scripts/Muon/GUI/Common/plot_widget/model_fit_pane/plot_model_fit_pane_presenter.py

```diff
@@ -0,0 +1,47 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+from mantidqt.utils.observer_pattern import GenericObserverWithArgPassing
+from Muon.GUI.Common.plot_widget.fit_pane.plot_fit_pane_presenter import PlotFitPanePresenter
+
+# A 10% margin when updating the x range based on the data x limits
+X_MARGIN = 0.1
```

Comment:
this should be in the context like the y margin:
https://github.com/mantidproject/mantid/blob/master/scripts/Muon/GUI/Common/contexts/plotting_context.py#L34